src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -0,0 +1,221 @@
+package org.onebusaway.alexa;
```

Comment:
@philipmw Can you please add the Apache license header to this and other new files?  For modified files, please add your name to copyright attribution list in the existing headers.